backend/inc/fnc.mipforms_core.php

```diff
@@ -172,6 +172,9 @@ function mip_forms_abstrakt($in, $input)
 		$spaces .= sprintf($mip_forms_tab['tab'], '&nbsp;');
 	}
 
+    $mip_forms_tab['left_width'] = empty($mip_forms_tab['left_width']) ? 0 : $mip_forms_tab['left_width'];
```

Comment:
What does this part do?

Reply:
It is important that  $mip_forms_tab['left_width'] is initialised with an integer. PHP7 will miscalculate otherwise.